unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldController.cs

```diff
@@ -454,6 +455,12 @@ public void EnterEditMode()
             HUDController.i.builderInWorldMainHud.RefreshCatalogAssetPack();
         }
 
+        if (HUDController.i.taskbarHud != null)
+        {
+            HUDController.i.taskbarHud.SetExploreInteractable(false);
+            HUDController.i.taskbarHud.SetMoreTutorialInteractable(false);
+        }
+
```

Comment:
A little worried about the approach, if we add a new button, we should remember to block it here or we could use it.

Can we have a function like "SetBuilderInWorldMode" and all buttons that are not explicitly marked as "work with builder" be desactivated?

Reply:
for example, the quest button, or the portable experience button

Reply:
Yeah, it is already planned. We'll implement in a separated PR 👍